frontend/src/redux/instructors/reducer.js

```diff
@@ -37,7 +37,7 @@ const instructorsSlice = createSlice({
             })
             .addCase(getInstructorsAsync.fulfilled, (state, action) => {
                 state.getInstructors = REQUEST_STATE.FULFILLED
-                state.list = action.payload
+                state.list = action.payload.data
```

Comment:
@kaiqianyyang all the responses from the server is formatted like `{data: {important stuff}, error: {error messages}}`, just a reminder for you to add `.data` to add other reducers for them to work.